consensus/consensus_v2.go

```diff
@@ -29,6 +29,12 @@ func (consensus *Consensus) handleMessageUpdate(payload []byte) {
 		return
 	}
 
+	// handle malformed messages to prevent crash
+	if msg.GetConsensus() == nil {
```

Comment:
not sure if this is the right place because there is Mode logic and ViewChangeing logic as well, and also its checked here: 

https://github.com/harmony-one/harmony/blob/a1e75767e01fa4fb88c616ff0dfa7a7b6d70c82b/consensus/consensus_v2.go#L58

Maybe let's keep the PR as a hotfix - that is, add this only in the explding validatorSanityCheck , keep our change local in scope.

Reply:
agreed. fixed to handle malformed messages locally. plz review.